exchange/adapter_map_test.go

```diff
@@ -0,0 +1,29 @@
+package exchange
+
+import (
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"testing"
+)
+
+func TestAdapterList(t *testing.T) {
+	list := AdapterList()
+	adapterInList(t, openrtb_ext.BidderAppnexus, list)
```

Comment:
Feel free to make the `bidderMap` in `openrtb_ext/bidders.go` public, and loop through it here.

There's virtually no chance that anyone will remember to update this test when they add new bidders.